app/views/movies/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= @movie.title %>'s Show Page</h1>
+<p>Creation Year: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+
+<p>Actors in this Movie:</p>
+  <% @actors.each do |actor| %>
+    <ul>
+      <li><%= actor.name %></li>
+    </ul>
+<% end %>
+
+<p>Average Age: <%= @movie.average_age.round(2) %></p>
```

Comment:
Consider adding the `.round(2)` to the average_age method just to encapsulate all of the averaging logic there. You could argue that rounding is a presentation tool but generally we keep mathematical operations like that together in the model